api/src/bin/main.rs

```diff
@@ -77,19 +75,17 @@ async fn main() {
     // Load injectable types.
```

Comment:
Do we need to keep this commented code around?

Reply:
I think the config once is safe to delete.  The Authorizer was a guide stone to help me remember where to inject the Authorizer. I'd at least like to keep a TODO comment here so that I can load this back into human RAM more easily :sob: